packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -53,7 +56,7 @@ const GitLabScopingSearchResults = (props: Props) => {
     graphql`
       fragment GitLabScopingSearchResults_query on Query
         @argumentDefinitions(
-          projectsFirst: {type: "Int", defaultValue: 20}
+          projectsFirst: {type: "Int", defaultValue: 75}
```

Comment:
Is it safe to increase to that number?

Reply:
Yeah, I just remember we had this issue. If we have refactor issue, I think it is fine